<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ytdl</class>
 <widget class="QMainWindow" name="ytdl">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ytdl</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>611</width>
      <height>121</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Music">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Music&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Music&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <attribute name="title">
      <string>Music</string>
     </attribute>
     <widget class="QLabel" name="labelQuality">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Quality:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMFormat">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Format:</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>481</width>
        <height>23</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="radiosMFormat">
       <item>
        <widget class="QRadioButton" name="radioAAC">
         <property name="text">
          <string>AAC</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">MFormatGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioFLAC">
         <property name="text">
          <string>FLAC</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">MFormatGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioMP3">
         <property name="text">
          <string>MP3</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">MFormatGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioOPUS">
         <property name="text">
          <string>OPUS</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">MFormatGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioWAV">
         <property name="text">
          <string>WAV</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">MFormatGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>501</width>
        <height>23</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="radiosQuality">
       <item>
        <widget class="QRadioButton" name="radioBest">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Best</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">MQualityGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioGood">
         <property name="text">
          <string>Good</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">MQualityGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioMedium">
         <property name="text">
          <string>Medium</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">MQualityGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioLow">
         <property name="text">
          <string>Low</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">MQualityGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Video">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Video</string>
     </attribute>
     <widget class="QLabel" name="labelResolution">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Max Resolution:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelVFormat">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Preferred Format:</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>451</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="radiosVFormat">
       <item>
        <widget class="QRadioButton" name="radio3GP">
         <property name="text">
          <string>3GP</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">VFormatGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioFLV">
         <property name="text">
          <string>FLV</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">VFormatGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioMP4">
         <property name="text">
          <string>MP4</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">VFormatGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioWEBM">
         <property name="text">
          <string>WEBM</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">VFormatGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>471</width>
        <height>23</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="radiosResolution">
       <item>
        <widget class="QRadioButton" name="radio4K">
         <property name="text">
          <string>4K</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">VResGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio1440p">
         <property name="text">
          <string>1440p</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">VResGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio1080p">
         <property name="text">
          <string>1080p</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">VResGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio720p">
         <property name="text">
          <string>720p</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">VResGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio480p">
         <property name="text">
          <string>480p</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">VResGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonBrowse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDownload">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineBrowse">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelURL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter URL:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineURL">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>501</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="VFormatGroup"/>
  <buttongroup name="MQualityGroup"/>
  <buttongroup name="MFormatGroup"/>
  <buttongroup name="VResGroup"/>
 </buttongroups>
</ui>
